examples/default/Makefile

```diff
@@ -31,7 +32,6 @@ QUIET ?= 1
 
 # Modules to include:
 USEMODULE += shell
-USEMODULE += shell_commands
```

Comment:
Why was this removed? oO

Reply:
Fixed in https://github.com/RIOT-OS/RIOT/pull/11599

Reply:
I have no idea how that sneaked in. I did touch about every example and test Makefile to update the `BOARD_INSUFFICIENT_MEMORY`, so I guess I just hit a wrong key and lost track of the changes :-(

Reply:
Me as well... Shit happens ;-).